### DISCUSSION TITLE
Customize Layout / Page Colors / Title / Logo

### DISCUSSION THREAD

**Discussion by drrescher** [iiab/iiab#3746] [Ideas]
What a nice Project :-)

One thing i'm thinking about which is totally irrelevant but somehow i care about:

Are there more ways of adjusting the users page layout than just the title font? Like I said, totally irrelevant for the project but I really dont like the colors on the page (the green and the blue). And I would like to change or just hide the Logo and maybe change the Title-text for a specific project.

Im working with a Raspberry Pi installation.

**Comment by tim-moody** [comment]
The short answer is no. I actually started work on some other color schemes, but wasn't very happy with any of them. I'm open to suggestions. I have roughed in a js function that changes the style and was planning to add more flexibility to Adm cons.

For obvious reasons I do not plan make the title configurable. I'd like to keep the branding.

**Reply by drrescher** [reply]
In the end its always personal preference.

I myself would keep it as simple as possible. I painted an example (in Grey, White, and a black Font - Like Wikipedia or the IIAB Website).
![Example](https://github.com/iiab/iiab/assets/139445589/99e24365-0c80-408b-9281-d62ee01bed14)

I'm not a programmer myself. But maybe just with custom HEX-Codes for each element would bring a lot of customization in my opinion.
